android/app/build.gradle

```
@@ -231,6 +238,38 @@ android {
             proguardFiles getDefaultProguardFile("proguard-android.txt"), "proguard-rules.pro"
         }
     }
+    flavorDimensions "version"
+
+    productFlavors {
+        prod {
+            minSdkVersion rootProject.ext.minSdkVersion
+            targetSdkVersion rootProject.ext.targetSdkVersion
+            resValue "string", "app_name", "ReactNativeBase"
+        }
+        develop {
+            applicationIdSuffix '.dev'
+            minSdkVersion rootProject.ext.minSdkVersion
+            targetSdkVersion rootProject.ext.targetSdkVersion
+            versionNameSuffix "- Develop"
+            resValue "string", "app_name", "RNBase - Develop"
```

What about this? This way will be more quicker for users to know which variant of the app they have installed

```suggestion
            resValue "string", "app_name", "Develop - RNBase"
```